raiden-dapp/src/components/navigation/Transfer.vue

```diff
@@ -251,129 +289,149 @@ export default class Transfer extends Mixins(BlockieMixin, NavigationMixin) {
     display: none;
   }
 
-  &__token-network-amount {
-    color: $color-white;
-    font-size: 24px;
-    font-weight: bold;
-    line-height: 19px;
-    margin: 20px 0 10px 0;
-    text-align: center;
-  }
+  &__menus {
+    background-color: $transfer-screen-color;
```

Comment:
Do we not want to use a color form the theme here? So isn't this a color to become re-used at multiple components?

Reply:
It was just introduced in this screen based on the mockups, not sure if we will be using it in other places later on.

Reply:
Okay let's try to be attentive and hope that we'll recognize when this color appears in future mock-ups again.